src/sst/core/configGraph.h

```diff
@@ -283,6 +295,8 @@ class ConfigComponent : public SST::Core::Serialization::serializable {
 // typedef SparseVectorMap<std::string,ConfigLink> ConfigLinkMap_t;
 /** Map IDs to Components */
 typedef SparseVectorMap<ComponentId_t,ConfigComponent> ConfigComponentMap_t;
+/** Map names to Components */
+// typedef std::map<std::string,ConfigComponent*> ConfigComponentNameMap_t;
```

Comment:
Is this supposed to be commented out? Is it intended for future use?

Reply:
Yes, it's for future use.  Today, the name map is contained in the cpython code.  I'm planning to switch this map over to live in the ConfigGraph, which is really where it belongs.  When I started this, I realized there were more things that should move into the ConfigGraph, like keeping track of the next component id.  I decided to put the code cleanup off and just get the current functionality into level.